### DISCUSSION TITLE
array-contains-any operator should have a 30 clause limit (currently limited to 10)?

### DISCUSSION THREAD

**Discussion by SKempin** [invertase/react-native-firebase#7574] [Q&A]
Currently using a array-contains-any query but am continuing to hit a limit of 10 elements. However the latest Firestore docs state this should now have a limit of [30 equality clauses](https://firebase.google.com/docs/firestore/query-data/queries#in_not-in_and_array-contains-any)?

also see https://github.com/firebase/flutterfire/issues/11203#issuecomment-1613034820

```    
"@react-native-firebase/app": "^18.7.3",
"@react-native-firebase/auth": "^18.7.3",
"@react-native-firebase/firestore": "^18.7.3",
```

![Simulator Screenshot - iPhone 13 mini - 2024-01-17 at 19 17 20](https://github.com/invertase/react-native-firebase/assets/10877466/7c207282-d43a-4ab6-9ef6-cc7ed3b33505)

**Comment by mikehardy** [comment]
Indeed - I believe this was fixed and released --> #7543

released as 18.8.0 here https://github.com/invertase/react-native-firebase/blob/main/CHANGELOG.md#1880-2024-01-25
